==14131== Memcheck, a memory error detector.
==14131== Copyright (C) 2002-2005, and GNU GPL'd, by Julian Seward et al.
==14131== Using LibVEX rev 1471, a library for dynamic binary translation.
==14131== Copyright (C) 2004-2005, and GNU GPL'd, by OpenWorks LLP.
==14131== Using valgrind-3.1.0, a dynamic binary instrumentation framework.
==14131== Copyright (C) 2000-2005, and GNU GPL'd, by Julian Seward et al.
==14131== For more details, rerun with: -v
==14131== 
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- DWARF2 CFI reader: unhandled CFI instruction 0:18
--14131-- WARNING: unhandled syscall: 273
--14131-- You may be able to write your own handler.
--14131-- Read the file README_MISSING_SYSCALL_OR_IOCTL.
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B79C03: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B81B31: PyString_InternInPlace (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B83611: PyString_InternFromString (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8AFA7: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8F290: PyType_Ready (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B90BFA: PyType_Ready (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7B80D: _Py_ReadyTypes (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC7851: Py_InitializeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCC417: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B79C03: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B81B31: PyString_InternInPlace (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B83611: PyString_InternFromString (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8AFA7: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8F290: PyType_Ready (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B90BFA: PyType_Ready (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7B80D: _Py_ReadyTypes (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC7851: Py_InitializeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCC417: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B924BD: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B926D6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8F573: PyType_Ready (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7B84D: _Py_ReadyTypes (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC7851: Py_InitializeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCC417: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131==  Address 0x543B020 is 456 bytes inside a block of size 768 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B79C03: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8F433: PyType_Ready (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7B84D: _Py_ReadyTypes (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC7851: Py_InitializeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCC417: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71815: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B924B0: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B926D6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8F573: PyType_Ready (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7B84D: _Py_ReadyTypes (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC7851: Py_InitializeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCC417: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131==  Address 0x543B020 is 456 bytes inside a block of size 768 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B79C03: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8F433: PyType_Ready (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7B84D: _Py_ReadyTypes (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC7851: Py_InitializeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCC417: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B924B0: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B926D6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8F573: PyType_Ready (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7B84D: _Py_ReadyTypes (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC7851: Py_InitializeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCC417: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131==  Address 0x543B020 is 456 bytes inside a block of size 768 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B79C03: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8F433: PyType_Ready (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7B84D: _Py_ReadyTypes (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC7851: Py_InitializeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCC417: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B926BE: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8F573: PyType_Ready (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7B84D: _Py_ReadyTypes (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC7851: Py_InitializeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCC417: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131==  Address 0x543B020 is 456 bytes inside a block of size 768 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B79C03: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8F433: PyType_Ready (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7B84D: _Py_ReadyTypes (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC7851: Py_InitializeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCC417: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA6135: _PyExc_Init (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC791A: Py_InitializeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCC417: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131==  Address 0x5449020 is 8 bytes inside a block of size 26 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4BA6135: _PyExc_Init (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC791A: Py_InitializeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCC417: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC2236: PyMarshal_ReadObjectFromString (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC383D: PyMarshal_ReadLastObjectFromFile (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBD4FC: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBEF88: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B5981E: PyObject_CallFunction (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x5464020 is 776 bytes inside a block of size 1,384 free'd
==14131==    at 0x4A1FF63: realloc (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B70BA2: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B70C95: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC2097: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC17C8: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC1BDF: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC2217: PyMarshal_ReadObjectFromString (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC383D: PyMarshal_ReadLastObjectFromFile (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBD4FC: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBEF88: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC2236: PyMarshal_ReadObjectFromString (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC37F2: PyMarshal_ReadLastObjectFromFile (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBD4FC: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBEF88: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x5473020 is 456 bytes inside a block of size 1,384 free'd
==14131==    at 0x4A1FF63: realloc (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B70BA2: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B70C95: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC2097: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC17C8: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC1BEC: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC2217: PyMarshal_ReadObjectFromString (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC37F2: PyMarshal_ReadLastObjectFromFile (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBD4FC: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBEF88: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC3802: PyMarshal_ReadLastObjectFromFile (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBD4FC: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBEF88: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x5467020 is 216 bytes inside a block of size 568 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x521581E: (within /lib64/libc-2.5.so)
==14131==    by 0x4BBDC35: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFC46: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B79C03: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B79DCC: PyDict_SetItemString (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BD311D: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BD3812: initposix (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBF371: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBF5AD: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x547C020 is 1,248 bytes inside a block of size 2,144 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x51EAFBC: qsort (in /lib64/libc-2.5.so)
==14131==    by 0x4BD30D1: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BD3812: initposix (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBF371: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBF5AD: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC2236: PyMarshal_ReadObjectFromString (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC383D: PyMarshal_ReadLastObjectFromFile (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBD4FC: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBEF88: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC2236: PyMarshal_ReadObjectFromString (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC383D: PyMarshal_ReadLastObjectFromFile (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBD4FC: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBEF88: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BADAB2: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBED07: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x54D5020 is 360 bytes inside a block of size 568 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x5214EBA: fclose (in /lib64/libc-2.5.so)
==14131==    by 0x4BBEF93: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B87E70: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BADAA2: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBED07: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x54D5020 is 360 bytes inside a block of size 568 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x5214EBA: fclose (in /lib64/libc-2.5.so)
==14131==    by 0x4BBEF93: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B87E70: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BADA92: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBED07: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x54D5020 is 360 bytes inside a block of size 568 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x5214EBA: fclose (in /lib64/libc-2.5.so)
==14131==    by 0x4BBEF93: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B87E70: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BADA82: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBED07: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x54D5020 is 360 bytes inside a block of size 568 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x5214EBA: fclose (in /lib64/libc-2.5.so)
==14131==    by 0x4BBEF93: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB4F4: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x54DA020 is 776 bytes inside a block of size 1,184 free'd
==14131==    at 0x4A1FF63: realloc (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B70BA2: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B70C95: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8FF2: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B79C03: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAC580: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x54DE020 is 40 bytes inside a block of size 64 free'd
==14131==    at 0x4A1FF63: realloc (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B70BA2: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B70C95: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC2097: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC1C22: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC17C8: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC1BDF: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC17C8: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC1BDF: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC2217: PyMarshal_ReadObjectFromString (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC383D: PyMarshal_ReadLastObjectFromFile (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBD4FC: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BADAB2: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B87E39: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BADAA2: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBED07: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x54DD020 is 56 bytes inside a block of size 568 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x521581E: (within /lib64/libc-2.5.so)
==14131==    by 0x4BBDC35: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFC46: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B87E70: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BADA92: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B87E39: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BADAA2: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBED07: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x54DE020 is 40 bytes inside a block of size 64 free'd
==14131==    at 0x4A1FF63: realloc (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B70BA2: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B70C95: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC2097: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC1C22: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC17C8: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC1BDF: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC17C8: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC1BDF: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC2217: PyMarshal_ReadObjectFromString (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC383D: PyMarshal_ReadLastObjectFromFile (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBD4FC: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7DADC: PyObject_Realloc (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCCE63: _PyObject_GC_Resize (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6BD04: PyFrame_New (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAC847: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6C6E3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B5DCE9: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB0C0: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6C6E3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x54DE020 is 40 bytes inside a block of size 64 free'd
==14131==    at 0x4A1FF63: realloc (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B70BA2: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B70C95: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC2097: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC1C22: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC17C8: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC1BDF: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC17C8: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC1BDF: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC2217: PyMarshal_ReadObjectFromString (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC383D: PyMarshal_ReadLastObjectFromFile (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBD4FC: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B53A79: PyGrammar_AddAccelerators (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B5435B: PyParser_New (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B543D1: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5BE2: PyParser_SimpleParseStringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5E1B: PyRun_StringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x54E1020 is 0 bytes after a block of size 272 alloc'd
==14131==    at 0x4A1EB08: malloc (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4BCD860: _PyObject_GC_Malloc (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCD95B: _PyObject_GC_New (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7A5D0: PyDict_New (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA9ACC: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6C6E3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B5DCE9: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB0C0: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B54557: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5BE2: PyParser_SimpleParseStringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5E1B: PyRun_StringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x5529020 is 472 bytes inside a block of size 640 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B53A79: PyGrammar_AddAccelerators (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B5435B: PyParser_New (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B543D1: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5BE2: PyParser_SimpleParseStringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5E1B: PyRun_StringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B5452B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5BE2: PyParser_SimpleParseStringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5E1B: PyRun_StringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x54E1020 is 0 bytes after a block of size 272 alloc'd
==14131==    at 0x4A1EB08: malloc (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4BCD860: _PyObject_GC_Malloc (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCD95B: _PyObject_GC_New (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7A5D0: PyDict_New (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA9ACC: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6C6E3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B5DCE9: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB0C0: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2110: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x552C020 is 0 bytes inside a block of size 44 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4BAD88F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2110: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BADA27: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D61: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x552C020 is 0 bytes inside a block of size 44 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4BAD88F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2110: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7DADC: PyObject_Realloc (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B83B06: _PyString_Resize (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2C1C: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x5530020 is 10,608 bytes inside a block of size 12,024 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B54557: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5BE2: PyParser_SimpleParseStringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5E1B: PyRun_StringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7DADC: PyObject_Realloc (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B83B06: _PyString_Resize (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2C3F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x5530020 is 10,608 bytes inside a block of size 12,024 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B54557: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5BE2: PyParser_SimpleParseStringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5E1B: PyRun_StringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAFBD6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2CE6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x5530020 is 10,608 bytes inside a block of size 12,024 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B54557: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5BE2: PyParser_SimpleParseStringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5E1B: PyRun_StringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAFBE0: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2CE6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x5530020 is 10,608 bytes inside a block of size 12,024 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B54557: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5BE2: PyParser_SimpleParseStringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5E1B: PyRun_StringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAFBEA: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2CE6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x5530020 is 10,608 bytes inside a block of size 12,024 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B54557: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5BE2: PyParser_SimpleParseStringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5E1B: PyRun_StringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD88F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2110: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x5530020 is 10,608 bytes inside a block of size 12,024 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B54557: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5BE2: PyParser_SimpleParseStringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5E1B: PyRun_StringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD922: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2110: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x5530020 is 10,608 bytes inside a block of size 12,024 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B54557: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5BE2: PyParser_SimpleParseStringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5E1B: PyRun_StringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BADA27: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6B487: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC9F50: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC9F59: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B798D2: PyDict_DelItem (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7992D: PyDict_DelItemString (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA846C: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x5530020 is 10,608 bytes inside a block of size 12,024 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B54557: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5BE2: PyParser_SimpleParseStringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5E1B: PyRun_StringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B54557: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5BE2: PyParser_SimpleParseStringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5E1B: PyRun_StringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B54557: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5BE2: PyParser_SimpleParseStringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5E1B: PyRun_StringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAFBD6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2CE6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAFBD6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2CE6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAFBE0: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2CE6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAFBE0: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2CE6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAFBEA: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2CE6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAFBEA: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2CE6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD922: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2110: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD922: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2110: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B5452B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5BE2: PyParser_SimpleParseStringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5E1B: PyRun_StringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B5452B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5BE2: PyParser_SimpleParseStringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5E1B: PyRun_StringFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA4208: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BADA27: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBED07: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x5474020 is 112 bytes inside a block of size 544 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B87E70: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BADA82: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBED07: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6B2ED: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB64C: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x555D020 is 0 bytes inside a block of size 64 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8852: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8852: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x555D020 is 0 bytes inside a block of size 64 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8852: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7DADC: PyObject_Realloc (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCCE63: _PyObject_GC_Resize (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6BD04: PyFrame_New (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB5D9: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x555D020 is 0 bytes inside a block of size 64 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8852: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B79C03: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8F433: PyType_Ready (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7BD8E: PyObject_GenericGetAttr (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA9AC2: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x555D020 is 0 bytes inside a block of size 64 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8852: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBC1FD: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBD0C4: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBD218: PyArg_ParseTuple (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BD2B1D: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x5560020 is 128 bytes inside a block of size 456 free'd
==14131==    at 0x4A1FF63: realloc (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4BCCE63: _PyObject_GC_Resize (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6BD04: PyFrame_New (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB5D9: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BD2CDB: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x5560020 is 128 bytes inside a block of size 456 free'd
==14131==    at 0x4A1FF63: realloc (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4BCCE63: _PyObject_GC_Resize (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6BD04: PyFrame_New (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB5D9: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6B2ED: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BACAC2: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x5561020 is 2,744 bytes inside a block of size 4,152 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x52468F8: closedir (in /lib64/libc-2.5.so)
==14131==    by 0x4BD2CD1: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BD11F1: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x5562020 is 144 bytes inside a block of size 704 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6B2ED: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BACAC2: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B79C03: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B79DCC: PyDict_SetItemString (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8F35D: PyType_Ready (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7BD8E: PyObject_GenericGetAttr (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA9AC2: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B79C03: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B79DCC: PyDict_SetItemString (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8F35D: PyType_Ready (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7BD8E: PyObject_GenericGetAttr (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA9AC2: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B65673: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B66960: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6B2ED: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BACAC2: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x5568020 is 13 bytes after a block of size 43 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4BD11F1: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBC1FD: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBC86C: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBCC01: PyArg_ParseTupleAndKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B67EF6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8A5C5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB0C0: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x556B020 is 8 bytes before a block of size 16 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B924B0: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B926D6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8F573: PyType_Ready (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7BD8E: PyObject_GenericGetAttr (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA9AC2: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B67F48: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8A5C5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB0C0: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x556B020 is 8 bytes before a block of size 16 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B924B0: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B926D6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8F573: PyType_Ready (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7BD8E: PyObject_GenericGetAttr (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA9AC2: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B65673: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B66C2A: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B66CFC: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA87D9: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x556B020 is 8 bytes before a block of size 16 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B924B0: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B926D6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8F573: PyType_Ready (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7BD8E: PyObject_GenericGetAttr (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA9AC2: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B66BB8: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B66CFC: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA87D9: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B66BB8: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B66CFC: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA87D9: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8852: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8852: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6B2ED: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB64C: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6B2ED: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB64C: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBC1FD: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBD0C4: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBD218: PyArg_ParseTuple (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BD1189: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBC1FD: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBD0C4: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBD218: PyArg_ParseTuple (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BD1189: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BD11F1: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BD11F1: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B66BB8: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B66CFC: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA87D9: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x559A020 is 5,800 bytes inside a block of size 8,192 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B65673: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B66960: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6B2ED: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BACAC2: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BD10A6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BD11DB: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB72E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x559F020 is 9,016 bytes inside a block of size 10,240 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B65673: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B66960: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6B2ED: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BACAC2: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B79C03: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBDE67: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFC46: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x55A1020 is 368 bytes inside a block of size 568 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x521581E: (within /lib64/libc-2.5.so)
==14131==    by 0x4BBDC35: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFC46: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B79C03: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAA097: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x55D5020 is 8 bytes after a block of size 272 alloc'd
==14131==    at 0x4A1EB08: malloc (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4BCD860: _PyObject_GC_Malloc (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCD95B: _PyObject_GC_New (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7A5D0: PyDict_New (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA9ACC: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B79672: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6B4A7: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BACAC2: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6C6E3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB8850: _PyCodec_Lookup (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB8BCA: PyCodec_Encoder (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC7AE8: Py_InitializeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCC417: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B79672: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6B4A7: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BACAC2: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6C6E3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB8850: _PyCodec_Lookup (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB8BCA: PyCodec_Encoder (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC7AE8: Py_InitializeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCC417: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7968A: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6B4A7: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BACAC2: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6C6E3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB8850: _PyCodec_Lookup (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB8BCA: PyCodec_Encoder (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC7AE8: Py_InitializeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCC417: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7968A: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6B4A7: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BACAC2: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6C6E3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB8850: _PyCodec_Lookup (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB8BCA: PyCodec_Encoder (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC7AE8: Py_InitializeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCC417: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B54557: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5C34: PyParser_SimpleParseFileFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5DAC: PyRun_FileExFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC6FFB: PyRun_SimpleFileExFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCCAA4: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131==  Address 0x5629020 is 16 bytes before a block of size 15 alloc'd
==14131==    at 0x4A1EB08: malloc (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x5229DA6: strdup (in /lib64/libc-2.5.so)
==14131==    by 0x4BC7AFC: Py_InitializeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCC417: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B5452B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5C34: PyParser_SimpleParseFileFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5DAC: PyRun_FileExFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC6FFB: PyRun_SimpleFileExFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCCAA4: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131==  Address 0x5629020 is 16 bytes before a block of size 15 alloc'd
==14131==    at 0x4A1EB08: malloc (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x5229DA6: strdup (in /lib64/libc-2.5.so)
==14131==    by 0x4BC7AFC: Py_InitializeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCC417: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD942: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2110: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC6FFB: PyRun_SimpleFileExFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCCAA4: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131==  Address 0x55D9020 is 7,912 bytes inside a block of size 8,192 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B5697C: PyTokenizer_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B5452B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5C34: PyParser_SimpleParseFileFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5DAC: PyRun_FileExFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC6FFB: PyRun_SimpleFileExFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCCAA4: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7968A: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD912: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2110: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC6FFB: PyRun_SimpleFileExFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCCAA4: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131==  Address 0x562C020 is 11,176 bytes inside a block of size 12,024 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B54557: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5C34: PyParser_SimpleParseFileFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5DAC: PyRun_FileExFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC6FFB: PyRun_SimpleFileExFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCCAA4: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD8E2: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2110: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC6FFB: PyRun_SimpleFileExFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCCAA4: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131==  Address 0x562C020 is 11,176 bytes inside a block of size 12,024 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B54557: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5C34: PyParser_SimpleParseFileFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5DAC: PyRun_FileExFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC6FFB: PyRun_SimpleFileExFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCCAA4: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2110: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC6FFB: PyRun_SimpleFileExFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCCAA4: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131==  Address 0x562C020 is 11,176 bytes inside a block of size 12,024 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4B54557: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5C34: PyParser_SimpleParseFileFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5DAC: PyRun_FileExFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC6FFB: PyRun_SimpleFileExFlags (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCCAA4: Py_Main (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x51D733E: __libc_start_main (in /lib64/libc-2.5.so)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B79C03: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B79DCC: PyDict_SetItemString (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC4895: Py_InitModule4 (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x57C5C56: init_opencv (opencv_wrap.cpp:63320)
==14131==    by 0x4BC1361: _PyImport_LoadDynamicModule (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x5639020 is not stack'd, malloc'd or (recently) free'd
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BADAB2: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBED07: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BADAB2: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBED07: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B79C03: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B79DCC: PyDict_SetItemString (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE112: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE1C7: initimp (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBF371: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBF5AD: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFFF: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0xA19A020 is 248 bytes inside a block of size 568 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x521581E: (within /lib64/libc-2.5.so)
==14131==    by 0x4BBDC35: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFAF4: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B77B77: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B79D00: PyDict_SetItem (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAC580: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0x56E3020 is not stack'd, malloc'd or (recently) free'd
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B87E39: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BADAA2: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBED07: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0xA1EB020 is 16 bytes before a block of size 840 alloc'd
==14131==    at 0x4A1EB08: malloc (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x4BCD860: _PyObject_GC_Malloc (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCD935: _PyObject_GC_NewVar (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B87FE6: PyTuple_New (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC1783: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC1BDF: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC2217: PyMarshal_ReadObjectFromString (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC383D: PyMarshal_ReadLastObjectFromFile (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBD4FC: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBEF88: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B79C03: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA9970: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0xA1F9020 is not stack'd, malloc'd or (recently) free'd
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BABBE9: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0xA1F9020 is not stack'd, malloc'd or (recently) free'd
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7A749: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6B2ED: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BACAC2: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B7A749: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6B2ED: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BACAC2: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAA9B2: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAA9B2: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6B2ED: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BACAC2: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71821: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6B2ED: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BACAC2: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== More than 100 errors detected.  Subsequent errors
==14131== will still be recorded, but in less detail than before.
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7DAE7: PyObject_Realloc (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCCE63: _PyObject_GC_Resize (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6BD04: PyFrame_New (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB5D9: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7DAF2: PyObject_Realloc (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BCCE63: _PyObject_GC_Resize (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6BD04: PyFrame_New (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB5D9: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B924BD: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B926D6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8F573: PyType_Ready (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0xA5B9EE2: (within /usr/lib64/python2.4/site-packages/numpy/core/multiarray.so)
==14131==    by 0xA5BA560: initmultiarray (in /usr/lib64/python2.4/site-packages/numpy/core/multiarray.so)
==14131==    by 0x4BC1361: _PyImport_LoadDynamicModule (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B924BD: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B926D6: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B8F573: PyType_Ready (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0xA5B9EE2: (within /usr/lib64/python2.4/site-packages/numpy/core/multiarray.so)
==14131==    by 0xA5BA560: initmultiarray (in /usr/lib64/python2.4/site-packages/numpy/core/multiarray.so)
==14131==    by 0x4BC1361: _PyImport_LoadDynamicModule (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Invalid read of size 4
==14131==    at 0x4B7D778: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6F14B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71E58: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B72174: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB570: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB62E: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB1D1: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==  Address 0xA4D5020 is 104 bytes inside a block of size 568 free'd
==14131==    at 0x4A1F6D2: free (in /usr/lib64/valgrind/amd64-linux/vgpreload_memcheck.so)
==14131==    by 0x521581E: (within /lib64/libc-2.5.so)
==14131==    by 0x4BBDC35: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFAF4: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6F14B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71E58: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAE061: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB0801: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB0BA0: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB1330: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2959: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0xB2F7CA8: initcPickle (in /usr/lib64/python2.4/lib-dynload/cPickle.so)
==14131==    by 0x4BC1361: _PyImport_LoadDynamicModule (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B6F14B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B71E58: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAE061: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB0801: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB0BA0: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB1330: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2959: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0xB2F7CA8: initcPickle (in /usr/lib64/python2.4/lib-dynload/cPickle.so)
==14131==    by 0x4BC1361: _PyImport_LoadDynamicModule (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7DAE7: PyObject_Realloc (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B83B06: _PyString_Resize (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2C1C: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB59A0: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB493F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB751D: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2C00: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0xB2F7CA8: initcPickle (in /usr/lib64/python2.4/lib-dynload/cPickle.so)
==14131==    by 0x4BC1361: _PyImport_LoadDynamicModule (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFFF: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7DAF2: PyObject_Realloc (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B83B06: _PyString_Resize (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2C1C: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB59A0: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB493F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB751D: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2C00: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0xB2F7CA8: initcPickle (in /usr/lib64/python2.4/lib-dynload/cPickle.so)
==14131==    by 0x4BC1361: _PyImport_LoadDynamicModule (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFFF: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD88F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2110: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB59A0: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB493F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB751D: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2C00: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0xB2F7CA8: initcPickle (in /usr/lib64/python2.4/lib-dynload/cPickle.so)
==14131==    by 0x4BC1361: _PyImport_LoadDynamicModule (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFFF: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD88F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2110: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB59A0: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB493F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB751D: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BB2C00: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC5D1B: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0xB2F7CA8: initcPickle (in /usr/lib64/python2.4/lib-dynload/cPickle.so)
==14131==    by 0x4BC1361: _PyImport_LoadDynamicModule (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFFF: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Conditional jump or move depends on uninitialised value(s)
==14131==    at 0x4B7D783: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC3802: PyMarshal_ReadLastObjectFromFile (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBD4FC: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBEF88: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFFF: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== Use of uninitialised value of size 8
==14131==    at 0x4B7D791: PyObject_Free (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC3802: PyMarshal_ReadLastObjectFromFile (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBD4FC: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBEF88: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFFF: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BC01AF: PyImport_ImportModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA485F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA7254: PyEval_CallObjectWithKeywords (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BA8D97: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
unhandled opc_aux = 0x7 
first_opcode == 0xDD
vex amd64->IR: unhandled instruction bytes: 0xDD 0x7C 0x24 0xFC
==14131== Your program just tried to execute an instruction that Valgrind
==14131== did not recognise.  There are two possible reasons for this.
==14131== 1. Your program has a bug and erroneously jumped to a non-code
==14131==    location.  If you are running Memcheck and you just saw a
==14131==    warning about a bad jump, it's probably your program's fault.
==14131== 2. The instruction is legitimate but Valgrind doesn't handle it,
==14131==    i.e. it's Valgrind's fault.  If you think this is the case or
==14131==    you are not sure, please let us know.
==14131== Either way, Valgrind will now raise a SIGILL signal which will
==14131== probably kill your program.
==14131== 
==14131== Process terminating with default action of signal 4 (SIGILL)
==14131==  Illegal opcode at address 0x506A9A8
==14131==    at 0x506A9A8: fetestexcept (in /lib64/libm-2.5.so)
==14131==    by 0xA6EEFC6: (within /usr/lib64/python2.4/site-packages/numpy/core/umath.so)
==14131==    by 0xA6F0198: (within /usr/lib64/python2.4/site-packages/numpy/core/umath.so)
==14131==    by 0xA6F4756: (within /usr/lib64/python2.4/site-packages/numpy/core/umath.so)
==14131==    by 0x4B58D13: PyObject_Call (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAB0C0: PyEval_EvalFrame (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD035: PyEval_EvalCodeEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BAD0F9: PyEval_EvalCode (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBE89B: PyImport_ExecCodeModuleEx (in /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBECD5: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFB2F: (within /usr/lib64/libpython2.4.so.1.0)
==14131==    by 0x4BBFFC3: (within /usr/lib64/libpython2.4.so.1.0)
==14131== 
==14131== ERROR SUMMARY: 1163 errors from 113 contexts (suppressed: 138 from 1)
==14131== malloc/free: in use at exit: 4,099,572 bytes in 1,721 blocks.
==14131== malloc/free: 5,609 allocs, 3,888 frees, 6,743,632 bytes allocated.
==14131== For counts of detected errors, rerun with: -v
==14131== searching for pointers to 1,721 not-freed blocks.
==14131== checked 7,910,568 bytes.
==14131== 
==14131== LEAK SUMMARY:
==14131==    definitely lost: 5,176 bytes in 4 blocks.
==14131==      possibly lost: 7,256 bytes in 11 blocks.
==14131==    still reachable: 4,087,140 bytes in 1,706 blocks.
==14131==         suppressed: 0 bytes in 0 blocks.
==14131== Use --leak-check=full to see details of leaked memory.
